extensions/wikia/Email/EmailController.class.php

```diff
@@ -232,6 +239,24 @@ protected function getBody() {
 	}
 
 	/**
+	 * Create a plain text version of the body HTML
+	 *
+	 * @param string $html The HTML for the email body
+	 *
+	 * @return string
+	 */
+	protected function bodyHtmlToText( $html ) {
+		$bodyText = strip_tags( $html );
+
+		// Get rid of multiple blank white lines
+		$bodyText = preg_replace( '/^\h*\v+/m', '', $bodyText );
+
+		// Get rid of leading spacing/indenting
+		$bodyText = preg_replace( '/^[\t ]+/m', '', $bodyText );
+		return $bodyText;
+	}
```

Comment:
Eh, I'm not convinced.  There really isn't anything in the acceptance criteria about how this happens or looks except 'no HTML.'  I'm not a big fan of tests where a failing test gets fixed simply by copying and pasting the new output to the accepted results of the test.